### DISCUSSION TITLE
v0.10.0-beta.5

### DISCUSSION THREAD

**Discussion by Samuel12321** [ShankarBUS/ModernFlyouts#1367] [General]
Maintenance release to restart development
## What's Changed
- Generated New Signing certificate
- updated .net to 9 preview
- fixed crash on launch
- integrated fixes from https://github.com/ModernFlyouts-Community/ModernFlyouts/pull/1217
- Updated Nuget Packages

* add GSMTC support status for `feeluown` by @mokurin000 in https://github.com/ModernFlyouts-Community/ModernFlyouts/pull/1261
* fix How-Tos link on README by @IsaacHatton in https://github.com/ModernFlyouts-Community/ModernFlyouts/pull/1306

## New Contributors
* @mokurin000 made their first contribution in https://github.com/ModernFlyouts-Community/ModernFlyouts/pull/1261
* @IsaacHatton made their first contribution in https://github.com/ModernFlyouts-Community/ModernFlyouts/pull/1306

**Full Changelog**: https://github.com/ModernFlyouts-Community/ModernFlyouts/compare/v0.10.0-beta.4...v0.10.0-beta.5

**Other:
Remember to install the certificate in the trusted root authority under "local machine". As this is a pre-release i haven't signed it via the MS Store.**

<hr /><em>This discussion was created from the release <a href='https://github.com/ModernFlyouts-Community/ModernFlyouts/releases/tag/v0.10.0-beta.5'>v0.10.0-beta.5</a>.</em>

**Comment by jh30uk** [comment]
Thank you.

**Comment by GibbyGibson1** [comment]
Can someone help me install this/ join your discord for more questions

**Comment by gamnull** [comment]
Is it possible to remove the gray border?
![Screenshot 2024-12-10 191740](https://github.com/user-attachments/assets/fb72e40e-9ddf-49c0-9743-f9e1f2d14b8e)

**Comment by Genz-RBX** [comment]
Thank god i found this, 0.9.3.0 on my pc(win11) was just showing invisible audio flyout

**Comment by vyrtuaPRO** [comment]
Even i install the certificate correcly windows return with error and ModernFlyouts doesnt show on superior bar. Any tips?

**Comment by GeorgeTheProgrammer01** [comment]
If anyone needs help with installation. 

Run the certificate first, Install Certificate > Local Machine > "Place all certificates in the following store" Look for "Trusted People"

Then install the Modern Flyout

**Reply by casperdcvd** [reply]
Thank you, I had tried multiple ways to install the certificate but this finally made it work. No more invisible ModernFlyouts!

**Reply by Foster13** [reply]
how can I got the certificate?

**Reply by mark2090** [reply]
Foster13:
Go below to the post made by me, (mark2090), made on July 21, 2025. 
My post below explains how to install the APP with the certificate.

**Reply by kppfzj** [reply]
> Foster13: Go below to the post made by me, (mark2090), made on July 21, 2025. My post below explains how to install the APP with the certificate.

Is there a fix for it not showing up on fullscreen apps?

**Reply by mark2090** [reply]
I am not sure what APPs you refer to that are full screen that seems to block the Modern Flyout from showing, but I tried it having my browser on full screen, playing a video, and all works fine. Tried it with my Windows Player APP and it works fine there too. 

The ModernFlyout APP itself does not have full screen functionality, it is one size only. 

What APP that is full screen that blocks the ModernFlyout from appearing?
ModernFlyout is a separate APP to all the others, it is not controlled or part of any other APP that provides volume control. It is connected to the main Windows volume control, so if you have a volume control on your keyboard, then the ModernFlyout APP appears. 

If what I said does not fit what problem your having, then you need to provide better and more details of what is happening on your computer with ModernFlyOuts, assuming you have it installed now. 

If you have not installed it, or have and its not working, then uninstall it completely, both the old and new, and reboot, then reinstall using the instructions given.

**Reply by kppfzj** [reply]
I see i was referring to for example when gaming, on fullscreen mode. Back before windows 11 it was able to display ontop of fullscreen mode however now it dosent work.

**Reply by manstheram** [reply]
The feature is gone for now, I didn't know how exactly to fix it again as that part of the code was not open source. If you want to fix it yourself, it may have used some sort of DLL injection.

**Reply by mark2090** [reply]
Ok, that makes a big difference.

A game when in full screen, (no window border at all), the game will not allow ModernFlyout to show, it will be hidden behind the full screen gaming, yet it still works.

If you make the gaming client windowed, but expand that window border full screen, still seeing the window border around the edges, then the ModernFlyOut will appear on top.

If you make the game window as  a smaller "Window", means the size is smaller than your full screen and you can see portions of your desktop behind it around the windows frame, then ModernFlyOut will appear on top.

It is the way fullscreen affects any APPs from showing on top. 

Essentially the gaming window, being full screen, no window border, will dominate the screen and places all other windows or opened APPs behind it. 

Now, there could be exceptions, depending on the APP, but that is what is occurring.
Hope that helps

**Comment by yarkiv** [comment]
Hi.   Thank you for this update.  it works very well. But after scan my PC.   My Antivirus delete  somes files and  warm me about  malwares 

after scan : dump:generic.dacic.3754.2f30bf4f


Warning on 0.10.5 exe : ->   Gen:Variant.Lazy.631262

So I unistall it   (so sad) 


I tried to use the previous  version  (no AV  warning with 0.10.4  exe)  , but I can't install it despite cert. installation.


is it possible to push a clean version on MS Store ?

**Comment by MNLierman** [comment]
Hey, I just wanted to comment and say that, while I trust you, I believe that installing the certificate as a Trusted Root Authority is a very bad security practice. The proper place to install the certificate would be **Trusted People**. This allows someone with administrator permissions to install something they are aware of but without introducing massive security vulnerabilities by adding a Trusted CA Authority Certificate. The implications of this could be catistrophic. Like I said, I trust you, and I appreciate your work on this, but **it would be much safer to recommend that everyone install the certificate in Trusted People** which is a sort of supervised trusted certificate store.

**Reply by GeorgeTheProgrammer01** [reply]
I appreciate your input but I'm not one of the developers/contributors to this. I just commented on how I was able to install it on my machine.

**Reply by manstheram** [reply]
I actually would suggest you do not install this app for a while. I contributed a while back. The app is currently being rewritten by some guy, and will be updated eventually (hopefully). This beta version may have some issues with it and was only made as a response to the overwhelming amount of issues relating to outdated certificate problems. This version already has a lot of workarounds in it, without testing so much, so I would not suggest using it in its current state, at least for the moment being.

**Reply by MNLierman** [reply]
> I appreciate your input but I'm not one of the developers/contributors to this. I just commented on how I was able to install it on my machine. 

I thought I was replying to the Release. At least that's where my GitHub app shows me that I am commenting. The person who posted this release is suggesting to everyone to install the certificate as a Root CA Certificate which is incredibly bad security practice. It looks like you also commented about installing the certificate in Trusted People which is the correct thing to do. I would venture to Guess that a lot of people are unaware of how to properly install certificates and the best standard practices for doing so. 

But anyways, my comment shows as a comment reply to the latest release. 

Also, for me, the reason why I'm using this is because it's the only thing I've been able to find that has a good looking caps lock, insert lock OSD message. I like being aware of when these keys are turned on as I don't like having to retype things like passwords. Even if retyping something only takes a couple of seconds.

**Reply by komidawi** [reply]
@MNLierman Could you please tell what software did you try for OSD messages feature? Then just in a few words tell why you didn't like it. 
AFAIK there's such a feature in Windows natively, also probably Dell and/or Asus have their apps capable of OSD (but probably at the same time being highly bloated).
Thanks.

**Reply by MNLierman** [reply]
Hi Dawid,

Since you asked, I'll give you the full answer to your question: I tried several OSD apps for modifier keys that I found on Google and Bing. Most of them were visually unappealing to me. I prefer a clean and modern aesthetic, and these apps often had sharp corners or outdated designs that clashed with the modern design of my new laptop and Win 11's UX. 

I even attempted to create my own solution using AutoHotKey, but I couldn't achieve the rounded corners or other factors that I wanted. I also explored options from Lenovo and other OEMs, but they either required a full software suite installation (ie. can't cherry pick the OSD function) or they wouldn't install on non *(brand)* PCs. One OEM app did work, but it was built on electron and consumed a significant amount of RAM – over 400mb – which was unacceptable to me.
 
Also, by default, Windows 11 itself does not inherently provide a universal OSD notification for modifier keys. Many OEMs (such as Dell, Lenovo, etc) and keyboard manufacturers include their own software that adds these OSDs. Therefore, if you see an OSD message in Windows and you didn't install any software yourself, then it's guaranteed due to pre-installed software from your PC's OEM or software provided by your keyboard manufacturer.

Ultimately, I found ModernFlyouts, and I'm very happy with it. I appreciate the developers' work in creating a free, visually appealing app that follows fluent/WinUI3 guidelines.

– Mike

**Reply by komidawi** [reply]
@MNLierman thank you very much for such a detailed response! 

I always really value first-hand experiences from actual user that faced the same problem rather than general random articles or one-word answers e.g. on reddit (especially nowadays when AI can write almost anything).

Your answer saves me time and effort then - I was about to do pretty the same.

Therefore it seems that ModerFlyouts is currently indeed the most viable solution, despite of its drawbacks, so we have to keep our fingers crossed for it to be developed and improved.

Kind regards,
komidawi

**Reply by MNLierman** [reply]
> @MNLierman thank you very much for such a detailed response! 

Yeah, no problem. I'm using the current beta version that was posted after installing the cert to Trusted People. (No one, and I repeat no one, should ever install unknown software into Trusted CAs or Root CAs, you are opening up a can of worms you cannot close.)

You can even hide the top bar on the notification flyouts, so that they match Windows 11 theme exactly. I centered mine to the bottom middle. 

While usable in it's current state just fine, I discovered that one could download the source code, and recompile the app without MS Store requirements. This was a recent discovery that I might be taking advantage of, as I hate installing things.

Some small modifications would have to be made, but it would take someone no longer than 5 minutes to do so. However, in the settings/manifest, the Win App SDK location needs to be referenced, as the MS Store handles that. I am really not a big fan of the MS Store push surrounding WinUI3 when using the corresponding project templates in Visual Studio – which is the reason why we are seeing so many developers use msixbundles now. 

For one, certificates have to be valid, and if they are not valid, the only option is to recompile the app from source code with a new self-signed certificate. No amount of Dev options or command line parameters will allow someone to install an unsigned or expired certificate. I tried for several hours. So stupid. The command line argument is literally there, but when supplied, a new error message is provided by windows, which basically says, f off, you need this signed. 

Microsoft thinks they know better than anyone else, right? With a traditional app, you can copy the folder, and run it on any computer. The compile option without the store is there. Very unnecessary to push so hard on the MS Store.

**Comment by lomba481** [comment]
I installed the certificate as you suggested, but when I double-click the .exe file, nothing happens.

**Reply by lomba481** [reply]
Actually, now even the default Windows flyouts don’t show up.

**Reply by komidawi** [reply]
In my case simple restart fixes this problem.  

In case you don't see anything in the tray, do the following:
`Ctrl + Alt + Delete` → `Task Manager`, Find `Modern Flyouts`, `Right Click` on it → `End Task`.  
Wait until it disappears, wait a few seconds, run it again (e.g. from Windows Start Menu).   
Check tray, `Right Click` → `Settings`, `General` → `Default Flyout` and set `Windows default` - then check if it works.  
Then try again checking `ModernFlyouts`.  
ModernFlyouts should now work properly.

Hope it helps!

**Reply by lomba481** [reply]
Unfortunately, my issue is different because ModernFlyouts doesn’t install at all. So, it doesn’t appear in Task Manager, and I can’t restart it as suggested. Do I need to click on the ModernFlyoutsHost_x64.exe file to install it? Do you have any other solutions I could try?

**Reply by komidawi** [reply]
@lomba481 Try to install it from this file: https://github.com/ModernFlyouts-Community/ModernFlyouts/releases/download/v0.10.0-beta.5/ModernFlyouts.Package_0.9.8622.0_x86_x64_arm64.msixbundle

**Comment by kppfzj** [comment]
hello, anyone having issues with it not popping up in fullscreen mode in games??  it will only pop up when I use windowed borderless. Is there any way to fix this. i really miss the pop up

**Reply by komidawi** [reply]
Probably it doesn't have such a feature. "Normal" Windows pop-ups also won't be visible while playing games on fullscreen mode

**Reply by kppfzj** [reply]
It did used to have this feature in the original version of modernflyouts. I guess the newer one dosent have it

**Comment by tsxsh** [comment]
yo i actually need a help to whenever i try to change song through the modern flyout thingy    it doesnt show up when im in a game

**Reply by kppfzj** [reply]
Im not sure but it wont work in fullscreen, if your in windowed screen it will show up. Thats all i got, I dont know how to fix it

**Comment by RafaeII0** [comment]
people who can help? I have a problem with the installation, I installed the certificate as indicated in the guide, but when I run the installer/program, it asks for a certificate and does not give me access to the download

**Reply by mark2090** [reply]
There are two downloads from Github for this APP, modernflyouts.
They are:
ModernFlyouts.Package_0.9.8622.0_x86_x64_arm64.msixbundle
ModernFlyouts.Package_0.9.8622.0_x86_x64_arm64.cer

Both above are for beta _version 10.0_. The developer has not yet finished redeveloping the APP. 
The beta version works fine,

To install the newer beta version, I had to do the following first:

- Uninstall the current Modernflyouts APP (per Win 11): 
WIN > Settings > Apps > Installed Apps > Modern Flyout - 3 dots, - click uninstall.
- Restart the system to unload any drivers that may be sitting in the memory.

For more information in uninstalling, and other troubleshooting methods, go to the Q&A page here:
[https://github.com/ModernFlyouts-Community/ModernFlyouts/wiki/How-Tos](https://github.com/ModernFlyouts-Community/ModernFlyouts/wiki/How-Tos)

Then do the following:

The second download above, with the extension "cer" is the certificate.
Install that first, - follow the instructions below per _Georgetheprogrammer01_, he said:

> Run the certificate first, Install Certificate > Local Machine > "Place all certificates in the following store" Look for "Trusted People"
> Then install the Modern Flyout

Double click the "cer" file like a regular program and windows will give a popup dialogue that you use to install the certificate as instructed by Georgetheprogrammer01. Then install the "msixbundle" file like any install program.

If you still have the older APP version after installing the newer (beta) version, (click "About" in the APP settings), that means you did not uninstall the older version correctly. Apparently, doing the uninstall via right-click the APP icon in Windows search does not work. Use the Windows Setting access (above) to uninstall the older version. 

Now it will work.

**Comment by vyrtuaPRO** [comment]
Im using the beta version since a few months... and i identified some bug with beta version: every time that windows starts, the app cause restart on my monitor (Samsung Odyssey G5 27" 2025).

When i deactivate the app, the monitor doesnt restart.

**Reply by mark2090** [reply]
There is very little information for anyone to get an idea of what may be happening.

I suggest the following:

- Uninstall the ModeryFlyout App.
- Then reinstall any drivers for the Monitor, and ensure that works fine.
- Reboot
- Reinstall Modern Flyouts, the current Beta version : https://github.com/ModernFlyouts-Community/ModernFlyouts/releases/tag/v0.10.0-beta.5
- Make sure the settings prevent the App from booting with Windows.
- Reboot
- When Windows loads, check to see of the App is still off (shutdown, not started). 
- Then, as the App is off, test the monitor, if all is good, then manually start the App and see what occurs.
- If the monitor does not restart with the App that you had to manually start, then keep the App set to a manual start after Windows loads completely.
- If the monitor restarts irrespective, then submit a bug, in fact do so anyway, unless the monitor somehow restarts, even with the App off, then it is not the App.

Reason is sometimes drivers clash when loading all at the same time and sometimes all works fine if certain drivers are loaded only after Windows completely loads.

**Comment by Sparticuz** [comment]
Looks like someone has published an updated Flyout's replacement. Seems to be working well for me.

https://github.com/unchihugo/FluentFlyout

**Reply by vyrtuaPRO** [reply]
Thanks for helping us... its working here too.
